cpp-ch/local-engine/Storages/SubstraitSource/ReadBufferBuilder.cpp

```diff
@@ -69,6 +71,34 @@ namespace ErrorCodes
 
 namespace local_engine
 {
+template <class key_type, class value_type>
```

Comment:
put this class in package Common?

Reply:
temporally put here, can be moved to common when reuse scenario occurs

Reply:
OK